balances-tx-payment/src/tests.rs

```diff
@@ -1,21 +1,20 @@
-use crate::balance_to_community_balance;
+use crate::{balance_to_community_balance, ONE_MICRO_KSM};
 use rstest::*;
 
-/// 1 micro KSM with 12 decimals
-const ONE_MICRO_KSM: u128 = 1_000_000;
-
 /// 1 community currency token with 18 decimals
 const ONE_CC: u128 = 1_000_000_000_000_000_000;
```

Comment:
I don't get this. If we have BalanceType U64F64 this means one cc is `1 << 64` not `1_000_000_000_000_000_000`
This needs more explaining comments, even if the new constants already improve readability

Reply:
@pifragile Why did you arbitrarily chose to set ONE_CC=1_000_000_000_000_000_000? I get that you tried to pick a number that avoids overflow, but your choice has nothing to do with the real interpretation of "one CC". We don't need a power of 10, we need the "real" ONE_CC. 

Or: why didn't you just set `ONE_CC: u128 = 1 << 64`? Then we could typecast this value to U64F64 and would really get 1 CC

Reply:
@brenzi the rationale behind this is that i assumed that fungibles uses a system similar to Ethereum where numbers are represented as a large integer (like 256 or 128 bit) and a given amount of decimals.
See here: https://github.com/paritytech/substrate/blob/ded44948e2d5a398abcb4e342b0513cb690961bb/frame/support/src/traits/tokens/fungibles/metadata.rs#L29

So for example with 5 decimals, the number `x` will be represented as `x * 10^5`. I chose 18 decimals, hence this number.

Reply:
Ok, but that confirms my concern. We use fixpoint with a base of 2, not base of 10. Unless you can convince me otherwise, I'd like this changed to 
`ONE_CC: u128 = 1 << 64` throughout

Reply:
this only works if no client of the balances pallet ever uses the `decimals` field, because we will obviously not be able to put a value in that field if we use your apporach. i will do some research. what is the reason you would like to change this? to avoid the conversions between base 2 / base 64 numbers?